py-polars/polars/functions/repeat.py

```diff
@@ -112,13 +120,27 @@ def repeat(
     """
     if isinstance(n, int):
         n = F.lit(n)
-    value = parse_as_expression(value, str_as_lit=True)
+    value = parse_as_expression(value, str_as_lit=True, list_as_lit=True, dtype=dtype)
     expr = wrap_expr(plr.repeat(value, n._pyexpr, dtype))
     if eager:
         return F.select(expr).to_series()
     return expr
 
 
+# dtypes that have a reasonable one/zero mapping;
+# for anything more elaborate should use `repeat`
+_ones_zeros = {
```

Comment:
Can we put this in a RLU function?

Reply:
Ahh, good plan; no need to do unnecessary work on import👌 
Coming right up...

Reply:
Done